WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/StartingInfo/PromptProgressBar/ProductCreationAIPromptProgressBar.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+
+struct ProductCreationAIPromptProgressBar: View {
+    @Binding var text: String
+    @StateObject private var viewModel = ProductCreationAIPromptProgressBarViewModel()
+
+    var body: some View {
+        VStack(content: {
+            ProgressView(value: viewModel.status.progress)
+                .progressViewStyle(ProductCreationAIBarProgressStyle(color: viewModel.status.color))
+
+
+            HStack(spacing: 0, content: {
+                Text(viewModel.status.mainDescription)
+                    .font(.footnote)
+                    .fontWeight(.semibold) +
+                Text(viewModel.status.secondaryDescription)
+                    .font(.footnote)
+                    .fontWeight(.regular)
+
+                Spacer()
+            })
+            .animation(.easeIn, value: viewModel.status)
+            .padding(.top, Layout.padding/2)
+        })
+        .padding([.top, .bottom, .leading, .trailing], Layout.padding)
```

Comment:
This can be simplified as below. 
```suggestion
        .padding(Layout.padding)
```

Reply:
Done 49533ded0b1c9647f7c60722c67bca4182a055bc